lib/src/components/NxTextInput/NxTextInput.tsx

```diff
@@ -67,16 +69,14 @@ const NxTextInput = forwardRef<HTMLInputElement | HTMLTextAreaElement, Props>(
         onKeyPress: inputOnKeyPress
       });
 
-      const invalidTextElementWithTooltip = (
+      return (
         <NxTooltip open={isInvalid}
                    title={firstValidationError}
                    className="nx-tooltip--validation-error"
                    placement="top-end">
           { textElement }
```

Comment:
There's no longer really any reason to break out the textElement as a separate variable

Reply:
> There's no longer really any reason to break out the textElement as a separate variable

Put it back into one statement in 9e7cb880b09b7d8ba0e700a4e8f247329abe5618